### PR TITLE
Fixes #25802 - Add disk usage info on storage domain select for oVirt

### PR DESCRIPTION
**Pull request by bagasse**
Print info on storage space for oVirt Storage domains in select box.

Before:

screenshot_foreman_ovirt1

After:

screenshot_foreman_ovirt2

### CODE REVIEW

**Review by chris1984 — APPROVED** [review]
LGTM @bagasse can you make hound happy?

**Comment by chris1984** [comment]
[test katello]

**Comment by bagasse** [comment]
@chris1984, sorry, forgot to click `resolve`, but that was already done.

**Review by chris1984 — DISMISSED** [review]
@bagasse awesome, lgtm and all tests are passing :) @tbrisker can this be merged?

File: app/helpers/compute_resources_vms_helper.rb [hunk 1]
```diff
@@ -193,6 +193,10 @@ def ovirt_fake_vms_count
     params['start'].to_i + 1 + [@vms.length, params['length'].to_i].min
   end
 
+  def ovirt_storage_domains_for_select(compute_resource)
+    compute_resource.storage_domains.map { |sd| OpenStruct.new({ id: sd.id, label: "#{sd.name} (" + _("Available") + ": #{sd.available.to_i / 1073741824} GiB, " + _("Used") + ": #{sd.used.to_i / 1073741824} GiB)" }) }
```
Comment (tbrisker) [inline app/helpers/compute_resources_vms_helper.rb]: you could use e.g. `sd.available.to_i / 1.gigabyte` just to simplify readability here

**Review by tbrisker — COMMENTED** [review]
Looks good to me, minor comment inline. @orrabin or @shiramax - can you please also take a look?

File: app/helpers/compute_resources_vms_helper.rb (hunk 1 above)
Comment (bagasse) [reply app/helpers/compute_resources_vms_helper.rb]: Done.

**Comment by bagasse** [comment]
Up! :) some new on this ? Thanks !

**Review by tbrisker — APPROVED** [review]
Thanks @bagasse !

**Pull request merged**